gradio/cli.py

```diff
@@ -0,0 +1,14 @@
+import sys
+
+import gradio.deploy_space
+import gradio.reload
+
+
+def cli():
+    args = sys.argv[1:]
+    if len(args) == 0:
+        raise ValueError("No file specified.")
+    if args[0] == "--deploy":
```

Comment:
Can we call it `deploy` as opposed to `--deploy`? I think `--` is usually used for a flag modifier on a command

Reply:
hmm I did want to clearly distinguish between using the existing gradio command for launch gradio in "auto relaunch mode", e.g. `gradio run.py`. I suppose it's unlikely that there's a python file named "deploy". Or we could choose another command line name like "gradio-deploy" though that feels ugly.

Reply:
good point about conflicting with filenames. Ok let's keep as is with `--deploy`!

Reply:
Agree with @freddyaboulton. Using `--deploy` to indicate an action instead of a flag is very unconventional. I would prefer to use `gradio deploy` (chance of a collision is so low)

Reply:
I don't love `gradio deploy` but ok